tree/ntuple/v7/inc/ROOT/RNTupleDescriptor.hxx

```diff
@@ -1399,6 +1410,8 @@ public:
    const RNTupleDescriptor &GetDescriptor() const { return fDescriptor; }
    RNTupleDescriptor MoveDescriptor();
 
+   void CreateFromSchema(const RNTupleDescriptor &descriptor);
```

Comment:
To me this method name is a bit vague, perhaps something like `SetSchemaFromExisting`?